frontend/dockerfile/docs/experimental.md

```diff
@@ -9,9 +9,9 @@ You need to use `docker build` CLI instead of `buildctl` CLI mentioned in this d
 See [the `docker build` document](https://docs.docker.com/engine/reference/commandline/build/) for the usage.
 
 ## Use experimental Dockerfile frontend
-The features mentioned in this document are experimentally available as [`docker/dockerfile-upstream:experimental`](https://hub.docker.com/r/docker/dockerfile-upstream/tags/) image.
+The features mentioned in this document are experimentally available as [`docker/dockerfile:experimental`](https://hub.docker.com/r/docker/dockerfile/tags/) image.
 
-To use the experimental features, the first line of your Dockerfile needs to be `# syntax=docker/dockerfile-upstream:experimental`.
+To use the experimental features, the first line of your Dockerfile needs to be `# syntax=docker/dockerfile:experimental`.
 As the experimental syntaxes may change in future revisions, you may want to pin the image to a specific revision.
 
 See also [#528](https://github.com/moby/buildkit/issues/528) for further information about planned `docker/dockerfile` releases.
```

Comment:
nit: maybe we should add a link to 
https://docs.google.com/document/d/1wP2oPvXf3duwV92c0jClhyhuijyrC6TXuKC4PA1XjhY/edit?usp=sharing (and we can remove the link to #528)